.browserslistrc

```diff
@@ -0,0 +1,4 @@
+last 2 versions
```

Comment:
Each project using this package will need to create the `.browserslistrc` file in its root.